src/Components/test/testassets/BasicTestApp/HttpClientTest/CookieCounterComponent.razor

```diff
@@ -1,3 +1,5 @@
+@using Microsoft.AspNetCore.Components.WebAssembly.Http
```

Comment:
Do you think this import needs to be in `_Imports.razor`? Another alternative might be to move the extensions to `Microsoft.AspNetCore.Components.WebAssembly` instead, so it's more discoverable.

Reply:
Yeah, I'd definitely be fine with it going into `_Imports.razor` in the template. Then it doesn't have to occupy an especially prevalent namespace.